modules/oadp-self-signed-certificate.adoc

```diff
@@ -39,3 +39,29 @@ spec:
 ----
 <1> Specify the Base64-encoded CA certificate string.
 <2> The `insecureSkipTLSVerify` configuration can be set to either `"true"` or `"false"`. If set to `"true"`, SSL/TLS security is disabled. If set to `"false"`, SSL/TLS security is enabled.
+
+[id="oadp-using-ca-certificates-with-velero-command-aliased-for-velero-deployment_{context}"]
+== Using CA certificates with velero command aliased for velero deployment
+
+You might want to use the Velero CLI without installing it locally on your system.
+
+If you want to use the Velero CLI without installing it locally you can do so by using an aliased Velero command, such as the following command:
+
+[source,terminal]
+----
+alias velero='oc -n openshift-adp exec deployment/velero -c velero -it -- ./velero'
+----
+
+If you want to use CA certificate with this command, you can add a certificate to the Velero deployment by running the following commands:
+
+[source,terminal]
+----
+$ oc get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}' | base64 -d | oc exec -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt"
+----
+
+[source,terminal]
+----
+velero describe backup <backup-name> --details --cacert /tmp/your-cacert.txt
```

Comment:
If it's alias then it does not require namespace, however if it's non-alias then it won't work, so it's better imo add namespace as it will work for both.

```shell
velero -n openshift-adp describe backup <backup-name> --details --cacert /tmp/your-cacert.txt```

Reply:
Resolved with one NIT below `$ ` before command.